libs/gperftools/Makefile

```diff
@@ -0,0 +1,80 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gperftools
+PKG_VERSION:=
+PKG_RELEASE:=1
+
+#PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+#PKG_SOURCE_URL:=https://codeload.github.com/$(PKG_NAME)/$(PKG_NAME)/tar.gz/$(PKG_NAME)-$(PKG_VERSION)?
+#PKG_HASH:=83e3bfdd28b8bcf53222c3798d4d395d52dadbbae59e8730c4a6d31a9c3732d8
+
+PKG_SOURCE_URL:=https://github.com/gperftools/gperftools
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_DATE:=2023-07-22
+PKG_SOURCE_VERSION:=42dde1b883b29921ac89234369f88b133821690c
+PKG_MIRROR_HASH:=7040ea346dd3be5d86af97072254f0fc3fec17c33d0956c0eb961f4e449c9b8f
+
+PKG_MAINTAINER:=John Audia <therealgraysky@proton.me>
+PKG_LICENSE:=BSD-3-Clause
+PKG_LICENSE_FILES:=COPYING
+
+PKG_BUILD_FLAGS:=no-mips16
+PKG_INSTALL:=1
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/gperftools-headers
+  CATEGORY:=Libraries
+  SECTION:=libs
+  TITLE:=Gperftools Headers
+  URL:=https://github.com/gperftools/gperftools
+  DEPENDS:=@TARGET_x86_64
+endef
+
+define Package/gperftools-runtime
+  SECTION:=libs
+  CATEGORY:=Libraries
+  TITLE:=Gperftools Runtime
+  URL:=https://github.com/gperftools/gperftools
+  DEPENDS:=@TARGET_x86_64 +libc +libunwind +libgcc +libstdcpp
+endef
+
+define Package/gperftools-headers/description
+  This package contains the headers for gperftools.
+endef
+
+define Package/gperftools-runtime/description
+  This package contains the shared objects and bins for gperftools.
```

Comment:
This is kinda short description, though.

Reply:
I expanded this.